packages/graphback-cli/src/templates/configTemplates.ts

```diff
@@ -37,6 +37,14 @@ const generationConfig = {
   "disableGen": false
 }
 
+const paths = {
+  "model": "./model",
+  "generatedResolvers": "./src/resolvers",
+  "customResolvers": "./src/resolvers",
+  "schema": "./src/schema",
+  "client": "./client/src/graphql"
```

Comment:
This will throw error since `mkdir` creates folder if parent folder exists. To be precise the error
```
Error: ENOENT: no such file or directory, mkdir '/home/ankitj/apollo/client/src/graphql'
```
Are we assuming the user already has the folder structure created?

Reply:
I think we can create a structure if it is not there.

Reply:
hi there,
Ankit did you check this? We are first mkdir with recursive { true } which solves the issue.

Reply:
@MStokluska I checked a bit and seems to be an issue the node versioning. I had downgraded my node version to v9.1.1 for some reason. The recursive true option works for anything above v10